extensions/2.0/Khronos/KHR_behavior/README.md

```diff
@@ -62,6 +62,290 @@ From these requirements, the main features can be deducted:
+
+### Examples
+
+This example's first behavior opens the door when it is clicked, and closes it when clicked again. No need for two animations: it is simply played backward via the `speed` parameter. This also means it will operate naturally when clicked in rapid succession: opening partly, then closing again without discontinuity. 
+
+The second behavior indicates to the user they should try selecting the door by blinking it red when they get close to it and it is in view, or if they hover over it. This action will reoccur whenever the user moves away and returns. It uses `functions` to accomplish this by gating on the value of the e.g. proximity state. This is different than a `waitAll` because in that case the user would have to *both* move away and turn away, then come back before it would re-fire. Instead it will refire even when the user turns away and back again without moving.
+
+```json
+{
+  "functions":[
+    {// 0
+      "name": "Near Door",
+      "type": "query/proximity",
+      "parameters": {
+        "mesh": 5
+      }
+    },
+    {// 1
+      "name": "Looking at Door",
+      "type": "query/inView",
+      "parameters": {
+        "mesh": 5
+      }
+    }
+  ],
+  "actions":[
+    // Door activation behavior
+    {// 0
+      "name": "Activate Door",
+      "type": "active/select",
+      "parameters": {
+        "mesh": 5
+      }
+    },
+    {// 1
+      "name": "Open/Close",
+      "type": "stateful/flipflop",
+      "input": [
+        {
+          "action": 0,
+        }
+      ],
+      "parameters": {
+        "animation": 2
+      }
+    },
+    {// 2
+      "name": "Open Door",
+      "type": "stateful/animation",
+      "input": [
+        {
+          "action": 1,
+          "output": 0
+        }
+      ],
+      "parameters": {
+        "animation": 2
+      }
+    },
+    {// 3
+      "name": "Open Door Sound",
+      "type": "stateless/playSound",
+      "input": [
+        {
+          "action": 1,
+          "output": 0
+        }
+      ],
+      "parameters": {
+        "audio": 0
+      }
+    },
+    {// 4
+      "name": "Close Door",
+      "type": "stateful/animation",
+      "input": [
+        {
+          "action": 1,
+          "output": 1
+        }
+      ],
+      "parameters": {
+        "animation": 2,
+        "speed": -1
+      }
+    },
+    {// 5
+      "name": "Close Door Sound",
+      "type": "stateless/playSound",
+      "input": [
+        {
+          "action": 1,
+          "output": 1
+        }
+      ],
+      "parameters": {
+        "audio": 1
+      }
+    },
+    // Door indication behavior
+    {// 6
+      "name": "Approach Door",
+      "type": "passive/proximityStart",
+      "parameters": {
+        "mesh": 5,
+        "distance": 2
+      }
+    },
+    {// 7
+      "name": "At Door",
+      "type": "stateless/gate",
+      "input": [
+        {
+          "action": 6
+        }
+      ],
+      "parameters": {
+        "if": {
+          "function": 1
```

Comment:
Just like the `$operation` syntax below. I'm using different names mostly to disambiguate until we come to consensus.